dockerfiles/ci/buster/php-8.0/configure_shared_ext.sh

```diff
@@ -7,9 +7,10 @@ ${PHP_SRC_DIR}/configure \
     --disable-all \
     --enable-cgi \
     --enable-fpm \
-    --enable-pcntl \
+    --enable-pcntl=shared \
```

Comment:
The reason we aren't using this method to install ext/curl (for example) is to mimic the behavior of some platforms that just copy over the `.so` without doing a `make install`. That way we can test the scenario where the curl headers aren't installed but curl is still loaded as a shared lib.

I don't see any headers being installed for ext/pcntl or ext/ffi in their config.m4 files, so I think we should be OK to use this technique for these two exts (assuming that doesn't change in the future). But can you think of any other differences for using this technique vs copying over the `.so` file without running `make install`?

Reply:
Exactly, in this scenario it does not make any difference. Otherwise I'd moved it into the post install step there.